packages/ai-llamafile/src/common/llamafile-server-manager.ts

```diff
@@ -0,0 +1,29 @@
+// *****************************************************************************
+// Copyright (C) 2024 EclipseSource GmbH.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
+// *****************************************************************************
+export const LlamafileServerManager = Symbol('LlamafileServerManager');
+
+export const LlamafileServerManagerPath = '/services/llamafileservermanager';
+
+export interface LlamafileServerManager {
```

Comment:
This could be adapted to be the `LlamafileManager` and also handle the language models on the backend.